Exec/RegTests/CMakeLists.txt

```diff
@@ -26,7 +26,7 @@ if(PELEC_ENABLE_AMREX_EB)
   add_subdirectory(EB-C10)
   add_subdirectory(EB-C11)
   add_subdirectory(EB-C12)
-  add_subdirectory(EB-C14)
+  #add_subdirectory(EB-C14)
```

Comment:
Is this intended?

Reply:
Yeah EB-C14 test is disabled due to FPEs so I figure why bother building it.

Reply:
oh right